packages/block-library/src/common.scss

```diff
@@ -81,3 +81,7 @@
 .items-justified-space-between {
 	justify-content: space-between;
 }
+
+img {
+	height: auto;
```

Comment:
Not sure I understand the issue and fix here. I tried using an image and it didn't seem stretched for me. Maybe there are special cases? Anyway, it seems this is overriding something that comes from Core itself. Is that style broken? (inline image height) Should we remove it in Core itself directly. 

cc @felixarntz as it seems related to your work on lazy loading images.

Reply:
It depends on the proportions of your image. You can see it in https://maylandblockstarter.wordpress.com/

Reply:
This is indeed not great, it seems we shouldn't be including that height in the first place, shouldn't we?

Reply:
I can reproduce this for image blocks when testing the `emptytheme` when:

- There's a wide or full alignment selected.
- The image has "Full Size" set as [the image size](https://cloudup.com/c-F5ZllM3Uj). 

When an image block is inserted, it gets some height and width by default. But when Gutenberg adjusts those, things get stretched. 

---

**Default (Large Size)**

Editor|Front
---|--- 
![Screen Shot 2021-03-22 at 12 08 17 PM](https://user-images.githubusercontent.com/1202812/112021367-9b366f80-8b07-11eb-8d3b-b65457b08363.png)|![Screen Shot 2021-03-22 at 12 11 04 PM](https://user-images.githubusercontent.com/1202812/112021486-b99c6b00-8b07-11eb-92dd-1c6057e661aa.png)

**Wide (Large Size)**

Editor|Front
---|--- 
![Screen Shot 2021-03-22 at 12 09 10 PM](https://user-images.githubusercontent.com/1202812/112021520-c3be6980-8b07-11eb-9edf-1b64bd417dd2.png)|![Screen Shot 2021-03-22 at 12 09 18 PM](https://user-images.githubusercontent.com/1202812/112021522-c3be6980-8b07-11eb-9d5b-f43265764936.png)

**Default (Full Size)**

Editor|Front
---|---
![Screen Shot 2021-03-22 at 12 14 50 PM](https://user-images.githubusercontent.com/1202812/112022007-40514800-8b08-11eb-82ff-617cbd7f35c3.png)|![Screen Shot 2021-03-22 at 12 15 04 PM](https://user-images.githubusercontent.com/1202812/112022011-40e9de80-8b08-11eb-81c8-deb5588ccba8.png)

Reply:
> but it probably shouldn't need to be fixed in the themes

I agree with this yes

Reply:
In principle, IMO this is the responsibility of themes to provide. Themes supporting the block editor have the responsibility of providing styles for `.alignwide` and `.alignfull`. As part of that (and as part of any usage that makes images span a certain width), it's necessary that any image has its height reset. So I'd argue, every theme should e.g. include something like
```css
img.alignwide,
img.alignfull {
    height: auto;
}
```

I agree though that it makes sense to solve this in Gutenberg because a theme developer may not be aware of WordPress images potentially having size attributes, and it is a trivial fix. We're certainly solving this widely by including it in Gutenberg itself.

Reply:
> In principle, IMO this is the responsibility of themes to provide.

This goest against the principles of Full Site Editing. In Full Site Editing, the theme doesn't have to provide anything for things to work properly and to have the editor match the frontend. You can start with an empty theme, go to the site editor and start building templates without having to provide any custom CSS.

Based on this principle, I don't think it makes sense anymore to have these inline attributes (the height at least) anymore in WordPress. There are just too many situations where things could go wrong. Thing just random images inside columns, inside a small "group" block. Images widths are not going to match the actual image width in a lot of situations, responsive mode is another one too.

So I'm not sure it makes sense to add these inline attributes if we override the styles later in a stylesheet.

Reply:
>  Themes supporting the block editor have the responsibility of providing styles for .alignwide and .alignfull. 

This is no longer the case for themes that opt into the new alignments offered in FSE (https://github.com/WordPress/gutenberg/pull/29335)